doc/Analytics.xml

```diff
@@ -4453,6 +4453,131 @@ xmlns:acme="http://www.acme.com/schema"&gt;
     </tt:ParentTopic>
   </tt:Messages>
 </tt:RuleDescription>
+]]></programlisting>
+    </section>
+    <section>
+      <title>Object Track Aggregation</title>
+      <para>This event is intended to describe an object throughout visible duration within a scene.
+        The description should include aggregated or summarized information along with contextual
+        data, such as images or bounding boxes, to assist clients in visualizing the object's
+        trajectory. This event can complement per-frame, per-object scene description by providing a
+        higher-level representation of the object’s trajectory, enhancing forensic analysis while
+        optimizing bandwidth consumption.</para>
+      <para>It is recommended to enable ObjectTrack data in sparsely crowded scenes to optimize data
+        produced from the device.</para>
+      <para>The process by which device aggregates object track data for a given object is out of
+        scope of this specification.</para>      
+      <para>The Object Track Aggregation rule generates an object track aggregation event for below
+        listed scenarios </para>
```

Comment:
@bsriramprasad, does the list below represent the scenario mentioned here? Starting the scenarios with 'Optionally' feels a bit unusual.

Reply:
Will update that paragraph a bit more for clarity.